helm/cluster-cloud-director/values.yaml

```diff
@@ -134,6 +134,26 @@ cluster:
                       name: 10-static-routes-dependency.conf
                   enabled: true
                   name: systemd-networkd.service
+                - contents:
+                    install:
+                      wantedBy:
+                        - multi-user.target
+                    service:
+                      additionalFields: |-
+                        {{- if $.global.connectivity.network.hostEntries -}}
+                        # Helm templating is a nightmare
```

Comment:
i don't like it, but i refuse to be defeated by templating

Reply:
I like it